tests/functional/backend/corpora/test_revisions.py

```diff
@@ -181,3 +174,9 @@ def test_revision_flow(self):
                     break
                 time.sleep(1)
             self.assertEqual(final_status_code, desired_status_code)
+
+    @retry(wait=wait_fixed(1), stop=stop_after_attempt(20))
```

Comment:
Retries a GET on the schema waiting 1 second between retries and stops after 20 tries.